rpmconf/Install/Util/globals.sh

```diff
@@ -59,7 +59,7 @@ zimbra-network-modules-ng"
 
 MYDIR="$(CDPATH= cd "$(dirname "$0")" && pwd)"
 if [ "$(cat ${MYDIR}/.BUILD_TYPE)" == "NETWORK" ]; then
-   OPTIONAL_PACKAGES="${OPTIONAL_PACKAGES} zimbra-modern-ui zimbra-modern-zimlets zimbra-patch zimbra-mta-patch zimbra-proxy-patch"
+   OPTIONAL_PACKAGES="${OPTIONAL_PACKAGES} zimbra-modern-ui zimbra-modern-zimlets zimbra-patch zimbra-mta-patch zimbra-proxy-patch zimbra-ldap-patch"
```

Comment:
@jiteshsojitra you might have to update us-tms with this change, specifically for multinode setup

Reply:
Done.

For e.g.

`INSTALL_PACKAGES="zimbra-core zimbra-ldap zimbra-logger zimbra-mta zimbra-snmp zimbra-store zimbra-apache zimbra-spell zimbra-convertd zimbra-memcached zimbra-proxy zimbra-archiving zimbra-patch zimbra-ldap-patch zimbra-proxy-patch zimbra-mta-patch additionalPackages"

`INSTALL_PACKAGES="zimbra-core zimbra-ldap zimbra-snmp zimbra-ldap-patch"`
`